Cargo.lock

```diff
@@ -840,25 +823,25 @@ checksum = "baf1de4339761588bc0619e3cbc0120ee582ebb74b53b4efbf79117bd2da40fd"
 
 [[package]]
 name = "chacha20"
-version = "0.7.3"
+version = "0.7.1"
```

Comment:
Are we downgrading this lib or I didn't get something? xd

Reply:
it's in a lockfile, so apparently this is the version that it resolves for upstream dependencies